react_main/src/pages/User/Profile.jsx

```diff
@@ -78,6 +80,14 @@ export default function Profile() {
                     setFriendsPage(1);
                     setStats(res.data.stats);
                     setGroups(res.data.groups);
+                    var videoMatches =  res.data.settings.youtube.match('v=([a-zA-Z0-9_-]+)&?');

```

Comment:
i didnt dig too much more into it just because this breaks if you don't have it set, should null coalesce this

Reply:
Oh, yeah.